### PR TITLE
Feature: Add confirm mobile screen

### PR DESCRIPTION
**Pull request by VladyslavP**

### CODE REVIEW

File: src/mobile/screens/ConfirmScreen/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
+import { selectUserInfo } from '../../../modules/user/profile';
+import { ConfirmScreen } from '../../../screens/ConfirmScreen';
+import { Modal } from '../../components/Modal';
+
+const ConfirmMobileScreen = () => {
+    const intl = useIntl();
+    const user = useSelector(selectUserInfo) || { level: 1 };
```
Comment (oyershov) [inline src/mobile/screens/ConfirmScreen/index.tsx:10]: 1. Default data for user already defined in redux reducer.
2. Default user level has not to be '1' in any case.

File: src/mobile/screens/ConfirmScreen/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                            <ConfirmScreen/>
+                        </Modal>
+                    </div>
+                    :
+                    <div className="cr-mobile-confirm__identity">
+                        <ConfirmScreen/>
+                    </div>
+            }
+        </div>
+    );
```
Comment (oyershov) [inline src/mobile/screens/ConfirmScreen/index.tsx:29]:
```suggestion
if (user.level === 1) {
       <div className="cr-mobile-confirm__phone">
            <Modal
                 isOpen
                 title={intl.formatMessage({id: 'page.body.profile.header.account.profile.phone.unverified.title'})}
            >
                  <ConfirmScreen/>
            </Modal>
       </div>
}

return (
    <div className="cr-mobile-confirm">
         <div className="cr-mobile-confirm__identity">
                <ConfirmScreen/>
         </div>
    </div>
);
```

File: src/components/UploadFile/index.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ interface OwnPropsTranslations {
     sizesText?: string;
     formatsText?: string;
     tipText?: string;
+    isMobileDevice?: boolean;
```
Comment (oyershov) [inline src/components/UploadFile/index.tsx:14]: Don path this as props. Use selector here

File: src/mobile/screens/ConfirmScreen/index.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
+import { defaultUser, selectUserInfo } from '../../../modules/user/profile';
+import { ConfirmScreen } from '../../../screens/ConfirmScreen';
+import { Modal } from '../../components/Modal';
+
+const ConfirmMobileScreen = () => {
+    const intl = useIntl();
+    const user = useSelector(selectUserInfo) || defaultUser;
```
Comment (oyershov) [inline src/mobile/screens/ConfirmScreen/index.tsx:10]: default user already defined from the redux reducer

File: src/components/UploadFile/index.tsx (hunk 3 above)
Comment (VladyslavP) [reply src/components/UploadFile/index.tsx:14]: it is a component, and it is not connected to redux

**Pull request merged**